dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/viewtools/JSONToolTest.java

```diff
@@ -0,0 +1,321 @@
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test {@link JSONTool}
+ * @author jsanca
+ */
+public class JSONToolTest extends IntegrationTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception{
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Method to test: {@link JSONTool#generate(String)}
+     * Given Scenario: Parsing an plain object
+     * ExpectedResult: Expected a map with the properties
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test
+    public void test_Generate_simple_Map() throws Exception {
+
+        final JSONTool jsonTool   = new JSONTool();
+        final String   stringJson = "{\n" +
+                "            \"body\": \"## Container: Blank Container\\r\\n## This is autogenerated code that cannot be changed\\r\\n#parseContainer('d71d56b4-0a8b-4bb2-be15-ffa5a23366ea','1539784124854')\\r\\n\",\n" +
+                "            \"canPublish\": true,\n" +
+                "            \"canRead\": true,\n" +
+                "            \"canWrite\": true,\n" +
+                "            \"categoryId\": \"9d11b327-9bf5-4d42-92bd-7894d56ed218\",\n" +
+                "            \"countAddContainer\": 0,\n" +
+                "            \"countContainers\": 0,\n" +
+                "            \"deleted\": false,\n" +
+                "            \"drawed\": false,\n" +
+                "            \"drawedBody\": null,\n" +
+                "            \"footer\": \"null\",\n" +
+                "            \"friendlyName\": \"Blank template (No: header, footer, javascript or css)\",\n" +
+                "            \"hasLiveVersion\": true,\n" +
+                "            \"headCode\": null,\n" +
+                "            \"header\": \"null\",\n" +
+                "            \"identifier\": \"7acdb856-4bbc-41c5-8695-a39c2e4a913f\",\n" +
+                "            \"image\": \"21ea6da6-68d0-48db-bce8-d1691abd6314\",\n" +
+                "            \"inode\": \"9d11b327-9bf5-4d42-92bd-7894d56ed218\",\n" +
+                "            \"live\": true,\n" +
+                "            \"locked\": false,\n" +
+                "            \"modDate\": 1570214794657,\n" +
+                "            \"modUser\": \"036fd43a-6d98-46e0-b22e-bae02cb86f0c\",\n" +
+                "            \"name\": \"Blank\",\n" +
+                "            \"new\": false,\n" +
+                "            \"owner\": \"null\",\n" +
+                "            \"selectedimage\": null,\n" +
+                "            \"showOnMenu\": true,\n" +
+                "            \"sortOrder\": 0,\n" +
```

Comment:
Codacy found an issue: [The String literal "            \"sortOrder\": 0,\n" appears 4 times in this file; the first occurrence is on line 69](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6315721)